### PR TITLE
Hackathon No.14

### PR DESCRIPTION
**Pull request by yangguohao**
MultiLabelSoftMarginLoss API 中文文档

PADDLEPADDLE_PR=41183

【Hackathon No.14】

### CODE REVIEW

File: docs/api/paddle/nn/MultiLabelSoftMarginLoss_cn.rst [hunk 1]
```diff
@@ -0,0 +1,40 @@
+.. _cn_api_paddle_nn_MultiLabelSoftMarginLoss:
+
+MultiLabelSoftMarginLoss
+-------------------------------
+
+.. py:class:: paddle.nn.MultiLabelSoftMarginLoss(weight:Optional=None, reduction: str = 'mean')
```
Comment (TCChenlong) [inline docs/api/paddle/nn/MultiLabelSoftMarginLoss_cn.rst]: 少了name

File: docs/api/paddle/nn/MultiLabelSoftMarginLoss_cn.rst [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    - **label** (Tensor) - :math:`[N, *]` ，标签 ``label`` 的维度、数据类型与输入 ``input`` 相同。
+    - **output** (Tensor) - 输出的Tensor。如果 :attr:`reduction` 是 ``'none'``, 则输出的维度为 :math:`[N, *]` , 与输入 ``input`` 的形状相同。如果 :attr:`reduction` 是 ``'mean'`` 或 ``'sum'``, 则输出的维度为 :math:`[1]` 。
+
+返回
+:::::::::
+   返回一个可调用计算Loss的类
+
+代码示例
+:::::::::
+COPY-FROM: Paddle.nn.layer.loss.MultiLabelSoftMarginLoss
```
Comment (TCChenlong) [inline docs/api/paddle/nn/MultiLabelSoftMarginLoss_cn.rst]: Paddle.nn.layer.loss.MultiLabelSoftMarginLoss -> paddle.nn.MultiLabelSoftMarginLoss
写公开调用的路径就好

File: docs/api/paddle/nn/functional/multi_label_soft_margin_loss_cn.rst [hunk 3]
```diff
@@ -0,0 +1,41 @@
+.. _cn_api_paddle_nn_functional_multi_label_soft_margin_losss:
+
+multi_label_soft_margin_loss
+-------------------------------
+
+.. py:class:: paddle.nn.MultiLabelSoftMarginLoss(weight:Optional=None, reduction: str = 'mean')
```
Comment (TCChenlong) [inline docs/api/paddle/nn/functional/multi_label_soft_margin_loss_cn.rst]: 这里写错了

File: docs/api/paddle/nn/functional/multi_label_soft_margin_loss_cn.rst [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    - **output** (Tensor) - 输出的Tensor。如果 :attr:`reduction` 是 ``'none'``, 则输出的维度为 :math:`[N, *]` , 与输入 ``input`` 的形状相同。如果 :attr:`reduction` 是 ``'mean'`` 或 ``'sum'``, 则输出的维度为 :math:`[1]` 。
+
+返回
+:::::::::
+   返回计算的Loss。
+
+
+代码示例
+:::::::::
+COPY-FROM: Paddle.nn.functional.loss.multi_label_soft_margin_loss
```
Comment (TCChenlong) [inline docs/api/paddle/nn/functional/multi_label_soft_margin_loss_cn.rst]: Paddle.nn.functional.loss.multi_label_soft_margin_loss -> paddle.nn.functional.multi_label_soft_margin_loss

File: docs/api/paddle/nn/MultiLabelSoftMarginLoss_cn.rst [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    - **reduction** (str,可选) - 指定应用于输出结果的计算方式，可选值有: ``'none'``, ``'mean'``, ``'sum'`` 。默认为 ``'mean'``，计算 Loss 的均值；设置为 ``'sum'`` 时，计算 Loss 的总和；设置为 ``'none'`` 时，则返回原始Loss。
+    - **name** (str，可选) - 操作的名称（可选，默认值为None）。更多信息请参见 :ref:`api_guide_Name` 。
+
+形状
+:::::::::
+    - **input** (Tensor) - :math:`[N, *]` , 其中N是batch_size， `*` 是任意其他维度。数据类型是float32、float64。
+    - **label** (Tensor) - :math:`[N, *]` ，标签 ``label`` 的维度、数据类型与输入 ``input`` 相同。
+    - **output** (Tensor) - 输出的Tensor。如果 :attr:`reduction` 是 ``'none'``, 则输出的维度为 :math:`[N, *]` , 与输入 ``input`` 的形状相同。如果 :attr:`reduction` 是 ``'mean'`` 或 ``'sum'``, 则输出的维度为 :math:`[1]` 。
+
+返回
```
Comment (TCChenlong) [inline docs/api/paddle/nn/MultiLabelSoftMarginLoss_cn.rst:40]: 这个不用写返回

File: docs/api/paddle/nn/functional/multi_label_soft_margin_loss_cn.rst [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+如果添加权重则再乘以对应的权重值
+
+
+最后，添加 `reduce` 操作到前面的输出Out上。当 `reduction` 为 `none` 时，直接返回最原始的 `Out` 结果。当 `reduction` 为 `mean` 时，返回输出的均值 :math:`Out = MEAN(Out)` 。当 `reduction` 为 `sum` 时，返回输出的求和 :math:`Out = SUM(Out)` 。
+
+
+参数
+:::::::::
+    - **weight** (Tensor，可选) - 手动设定权重，默认为None
```
Comment (TCChenlong) [inline docs/api/paddle/nn/functional/multi_label_soft_margin_loss_cn.rst:26]: 这里要写input&label

File: docs/api/paddle/nn/functional/multi_label_soft_margin_loss_cn.rst [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+    - **name** (str，可选) - 操作的名称（可选，默认值为None）。更多信息请参见 :ref:`api_guide_Name` 。
+
+形状
+:::::::::
+    - **input** (Tensor) - :math:`[N, *]` , 其中N是batch_size， `*` 是任意其他维度。数据类型是float32、float64。
+    - **label** (Tensor) - :math:`[N, *]` ，标签 ``label`` 的维度、数据类型与输入 ``input`` 相同。
+    - **output** (Tensor) - 输出的Tensor。如果 :attr:`reduction` 是 ``'none'``, 则输出的维度为 :math:`[N, *]` , 与输入 ``input`` 的形状相同。如果 :attr:`reduction` 是 ``'mean'`` 或 ``'sum'``, 则输出的维度为 :math:`[1]` 。
+
+返回
+:::::::::
```
Comment (TCChenlong) [inline docs/api/paddle/nn/functional/multi_label_soft_margin_loss_cn.rst:37]: 这里要写返回类型，比如
Tensor，

File: docs/api/paddle/nn/MultiLabelSoftMarginLoss_cn.rst (hunk 1 above)
Comment (yangguohao) [reply docs/api/paddle/nn/MultiLabelSoftMarginLoss_cn.rst]: 已修改

File: docs/api/paddle/nn/MultiLabelSoftMarginLoss_cn.rst (hunk 2 above)
Comment (yangguohao) [reply docs/api/paddle/nn/MultiLabelSoftMarginLoss_cn.rst]: 已修改

File: docs/api/paddle/nn/MultiLabelSoftMarginLoss_cn.rst (hunk 5 above)
Comment (yangguohao) [reply docs/api/paddle/nn/MultiLabelSoftMarginLoss_cn.rst:40]: 改为 返回计算 MultiLabelSoftMarginLoss 的可调用类。

File: docs/api/paddle/nn/functional/multi_label_soft_margin_loss_cn.rst (hunk 4 above)
Comment (yangguohao) [reply docs/api/paddle/nn/functional/multi_label_soft_margin_loss_cn.rst]: 已修改

File: docs/api/paddle/nn/functional/multi_label_soft_margin_loss_cn.rst (hunk 6 above)
Comment (yangguohao) [reply docs/api/paddle/nn/functional/multi_label_soft_margin_loss_cn.rst:26]: 已修改

File: docs/api/paddle/nn/functional/multi_label_soft_margin_loss_cn.rst (hunk 7 above)
Comment (yangguohao) [reply docs/api/paddle/nn/functional/multi_label_soft_margin_loss_cn.rst:37]: 已修改

File: docs/api/paddle/nn/functional/multi_label_soft_margin_loss_cn.rst (hunk 3 above)
Comment (yangguohao) [reply docs/api/paddle/nn/functional/multi_label_soft_margin_loss_cn.rst]: 已修改

**Review by TCChenlong — COMMENTED** [review]
@yangguohao 请解决一下冲突

**Review by TCChenlong — APPROVED** [review]
